modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/Countries.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClayList from '@clayui/list';
+import PropTypes from 'prop-types';
+import React, {useContext, useMemo, useState} from 'react';
+
+import {StoreStateContext} from '../../context/StoreContext';
+import {numberFormat} from '../../utils/numberFormat';
+
+const KEYWORD_VALUE_TYPE = [
+	{label: Liferay.Language.get('views'), name: 'views'},
+	{label: Liferay.Language.get('views-percentage'), name: 'views-percentage'},
+];
+
+export default function Countries({currentPage}) {
+	const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+	const [keywordValueType, setKeywordValueType] = useState(
+		KEYWORD_VALUE_TYPE[0]
+	);
+
+	const {languageTag, publishedToday} = useContext(StoreStateContext);
+
+	const countries = useMemo(() => {
```

Comment:
Why do we need to do this with the countries list?

Reply:
I think it's  not really necessary, I kept it because it was use with Keywords and wanted to see if i can keep it that way.  I can map directly the data from the currentPage object(line 123).